### PR TITLE
Add basic info to index.html

### PR DESCRIPTION
**Pull request by mollygraton**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,12 +7,27 @@
...
+          I enjoy biking & playing uke.
+      </p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <div id="photography">
+    </div>
+    <div id="talents">
+    </div>
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: Is this intended that there is no content in between?

**Review by cisxingg — CHANGES_REQUESTED** [review]
Looks good! Please don't merge until Warfa comments on this :)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (mollygraton) [reply portfolio/src/main/webapp/index.html]: Yep, was just creating the div layout for the page -- will add content in upcoming commits!

**Review by cisxingg — APPROVED** [review]

**Review by warfa-google — APPROVED** [review]
Looks good to me!

**Pull request merged**